Content.Server/Objectives/Systems/KillPersonConditionSystem.cs

```diff
@@ -86,11 +85,10 @@ private void OnHeadAssigned(EntityUid uid, PickRandomHeadComponent comp, ref Obj
         }
 
         var allHeads = new List<EntityUid>();
-        foreach (var mind in allHumans)
+        foreach (var minds in allHumans)
```

Comment:
`minds` here is a single entity, so it shouldn't be plural.
In these sorts of situations, my heuristic is to name it as a singular of the iterable's name, eg. `var human in allHumans`.